python/codepipeline-docker-build/app.py

```diff
@@ -0,0 +1,26 @@
+from aws_cdk import (
+    core,
+)
+
+from Base import Base
+from Pipeline import Pipeline
+
+
+# using props to pass in objects between stacks
+class Props():
+    def __init__(self):
+        self.namespace = 'cdk-example-pipeline'
+        self.region = 'us-east-1'
+
+
+props = Props()
+app = core.App()
+
+# stack for ecr, bucket, codebuild
+base = Base(app, f"{props.namespace}-base", props, )
```

Comment:
No need for the comma after `props` (the comma is only needed for multi-line calls).